<launch>
    <node name="hdt_arm_planning_node" pkg="hdt" type="hdt_arm_planning_node">
        <rosparam command="load" file="$(find hdt)/config/hdt_arm_planner.yaml"/>
        <rosparam command="load" file="$(find hdt)/config/hdt_collision_checking.yaml"/>
        <param name="action_set_filename" value="$(find hdt)/config/pr2.mprim"/>
        <!-- <param name="object_filename" value="$(find sbpl_arm_planner_test)/env/tabletop_hdt.env"/> -->
        <remap from="command" to="joint_path_command"/>
    </node>
</launch>
